ui/resizable.js

```diff
@@ -195,7 +196,11 @@ $.widget("ui.resizable", $.ui.mouse, {
 
 		this._renderAxis = function(target) {
 
-			var i, axis, padPos, padWrapper;
+			var i, axis, padPos, padWrapper, mouseDownHandlers = {
+				mousedown: function( event ) {
```

Comment:
And once this is simplified, we can just inline the object in the `_on()` call ;-)